### PR TITLE
Styling changes

### PR DESCRIPTION
**Pull request by ranbena**
- [x] Feature

## Description
ezgif com-video-to-gif

Added:
1. Transitioning bg color when dragging.
2. Box shadow to dragged element.
3. A few minor styling changes.

### CODE REVIEW

File: client/app/components/Parameters.less [hunk 1]
```diff
@@ -7,29 +7,50 @@
...
   background: white;
-  padding: 6px;
+  padding: 3px 6px 6px;
+
+  .parameter-container[data-draggable] & {
+    margin: 4px 0 0 4px;
+  }
+
+  // dragged element
+  body > & {
```
Comment (ranbena) [inline client/app/components/Parameters.less]: No other hook 😅

Comment (gabrieldutra) [reply client/app/components/Parameters.less]: It is possible to use `helperContainer` attr to append it somewhere else than the body, but I don't think it's worth it :)

Comment (gabrieldutra) [reply client/app/components/Parameters.less]: I wonder if `helperClass` is targeted on that element, I'll check that and edit this comment with my conclusion.

Comment (ranbena) [reply client/app/components/Parameters.less]: aaad44e

File: client/app/components/Parameters.less [hunk 2]
```diff
@@ -7,29 +7,49 @@
...
 .parameter-container {
   position: relative;
-  background-color: #f6f8f9;
-  display: inline;
-  padding: 35px 0 11px;
+
+  &[data-draggable] {
+    padding: 0 4px 4px 0;
+    transition: background-color 200ms ease-out;
+    transition-delay: 500ms; // short pause before returning to original bgcolor
```
[the remark this answers is not in the document]
Comment (ranbena) [reply client/app/components/Parameters.less]: You're right. Decreased fc54399

**Comment by ranbena** [comment]
> One last thing to be discussed: How should we handle long parameter titles (longer than the parameter width actually)?

I'm gonna reply to this in your PR.

**Pull request merged**